aioshelly/ble/__init__.py

```diff
@@ -105,3 +108,37 @@ async def async_ensure_ble_enabled(device: RpcDevice) -> bool:
     LOGGER.info("BLE enabled, restarting device %s:%s", device.ip_address, device.port)
     await device.trigger_reboot(3500)
     return True
+
+
+async def async_ble_supported(device: RpcDevice) -> bool:
+    """Check if BLE is supported.
```

Comment:
I think its a bit confusing to name it `ble_supported` as it doesn't show the intent of the function, which is checking if scripts are supported